README.md

```diff
@@ -27,16 +31,18 @@ virtual environment by running
 python -m pip install pandamonium
 ```
 
-### Install the old style of global scripts
+### Install oldschool way
 
-If you want to install the original version of `pandamonium` before it became a
-library and was a set of global level Python scripts you can still do that.
+If you can't get `pip` working you can just put the scripts you need
+in your path. Note that this is **deprecated** (but still works for
+now).
 
-1. Clone the repository at tag [`v0.1`][tag_v0.1]
+1. Clone the repository
```

Comment:
The idea here being to not add information on tags until things actually are deprecated?

Reply:
yup